samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp_ns.conf

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Multirole is the only currently supported role by SoftDevice.
+CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
+
+CONFIG_FPU=n
+CONFIG_PM=n
+CONFIG_HWINFO_NRF=n
+
+CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
```

Comment:
Why do we have to disable it? If this is necessary, can we have some comment describing the reason?

Reply:
Hmm, it seems that the code works without this one. Honestly, I've just copied it from the previous configurations because it was not supported some time ago, but now I will check whether we can remove this line from all places.

Reply:
Ok it is redundant and it works without it, I've removed it :)